core/templates/pages/topic-viewer-page/stories-list/topic-viewer-stories-list.component.html

```diff
@@ -1,10 +1,11 @@
 <div class="stories-list">
   <div class="oppia-stories-list-container">
-    <mat-card class="oppia-page-card oppia-long-text">
+    <mat-card class="oppia-page-card oppia-long-text" [ngClass]="checkMobileView() ? 'm-0': 'mat-card-margin-desktop'">
       <h2 class="tab-title">{{ topicName }}</h2>
       <div class="topic-description protractor-test-topic-description">
         {{ topicDescription }}
       </div>
+      <strong class="stories-can-play" *ngIf="checkMobileView()">Stories You Can Play</strong>
```

Comment:
Can you add a translation key for this? All learner facing text should be translated here and elsewhere.

Reply:
Done!